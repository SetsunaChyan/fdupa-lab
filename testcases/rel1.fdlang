function main(){
i = input();
j = i;

while(i > 0) {
    i = i - 1;
    j = j - 1;
}

check_interval(j, 0, 0);
}