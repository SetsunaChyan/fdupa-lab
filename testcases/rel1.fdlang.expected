Line 10: YES; j in [0, 0]
